aten/src/ATen/cpu/vec/sve/vec_bfloat16.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+// DO NOT DEFINE STATIC DATA IN THIS HEADER!
+// See Note [Do not compile initializers with AVX]
+
+#include <ATen/cpu/vec/intrinsics.h>
+#include <ATen/cpu/vec/vec_base.h>
+#include <c10/util/irange.h>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wignored-qualifiers"
+
+namespace at::vec {
+// See Note [CPU_CAPABILITY namespace]
+inline namespace CPU_CAPABILITY {
+
+#define CONVERT_NON_VECTORIZED_INIT(type, name) \
```

Comment:
for bfloat16, shouldn't we detect __ARM_FEATURE_SVE_BF16 and use some svcvt_bf16 variant?
for half, isn't svcvt_f32_f16 required in SVE?

Reply:
Thanks, I agree with your points and will try to implement your suggestion asap.

Reply:
Hi @swolchok 

Done by @Ryo-not-rio in PR https://github.com/pytorch/pytorch/pull/143666